tests/objc-cli/libmanaged/Tests/Tests.m

```diff
@@ -749,6 +749,11 @@ - (void) testOperatorOverloading {
 
 	XCTAssertTrue ([[Overloads_AllOperatorsWithFriendly decrementC1:oneFriend] value] == (1 - 1), "1 - 1 All");
 	XCTAssertTrue ([[Overloads_AllOperatorsWithFriendly incrementC1:oneFriend] value] == (1 + 1), "1 + 1 All");
+
+	Overloads_EqualOverrides * equalOverrideTwo = [[Overloads_EqualOverrides alloc] initWithVal:2];
+	Overloads_EqualOverrides * equalOverrideFour = [[Overloads_EqualOverrides alloc] initWithVal:4];
+
+	XCTAssertTrue ([equalOverrideTwo areEqual:equalOverrideFour] == NO, "2 != 4 EqualOverride");
```

Comment:
what needs testing is
`[Overloads_EqualOverrides areEqual:equalOverrideTwo x:equalOverrideFour] == NO`